src/defaultsettings.cpp

```diff
@@ -160,8 +160,9 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("generate_normalmaps", "false");
 	settings->setDefault("normalmaps_strength", "0.6");
 	settings->setDefault("normalmaps_smooth", "1");
-	settings->setDefault("parallax_occlusion_scale", "0.06");
-	settings->setDefault("parallax_occlusion_bias", "0.03");
+	settings->setDefault("parallax_occlusion_iterations", "4");
```

Comment:
The additional setting should be documented, and the changed defaults updated in `minetest.conf.example`